services/alertmanager/alertmanager.go

```diff
@@ -130,6 +131,47 @@ receivers:
 		err = ioutil.WriteFile(alertmanagerBaseConfigPath, []byte(defaultBase), 0644) //nolint:gosec
 		svc.l.Infof("%s created: %v.", alertmanagerBaseConfigPath, err)
 	}
+
```

Comment:
maybe this part should be extracted to a separate method and called right after `generateBaseConfig`?
it doesn't look like 100% depend on a code above.

Reply:
moved 5f791b0